pkg/apiserver/bootstrap.go

```diff
@@ -132,13 +132,17 @@ func bootstrap(params APIServerParams) error {
 
 	// if no status informers are provided, generate them from the helm release
 	informers := params.StatusInformers
-	if len(informers) == 0 && helm.IsHelmManaged() {
-		helmRelease, err := helm.GetRelease(helm.GetReleaseName())
-		if err != nil {
-			return errors.Wrap(err, "failed to get helm release")
-		}
-		if helmRelease != nil {
-			informers = appstate.GenerateStatusInformersForManifest(helmRelease.Manifest)
+	if informers == nil {
+		informers = []appstatetypes.StatusInformerString{}
+		if helm.IsHelmManaged() {
+			helmRelease, err := helm.GetRelease(helm.GetReleaseName())
+			if err != nil {
+				return errors.Wrap(err, "failed to get helm release")
+			}
+			if helmRelease != nil {
+				informers = appstate.GenerateStatusInformersForManifest(helmRelease.Manifest)
+				logger.Infof("Generated informers from Helm release: %v", informers)
```

Comment:
this log statement should be inside the `GenerateStatusInformersForManifest` function at the beginning, and be something like:
```
Generating status informers from Helm release
```